systems/olcf-frontier/system.py

```diff
@@ -0,0 +1,393 @@
+        f77: /usr/bin/gfortran-{self.compiler_version}
+        fc: /usr/bin/gfortran-{self.compiler_version}
+      operating_system: sles15
+      target: any
+      environment:
+        prepend_path: {}
+      extra_rpaths: []
+      modules:
+      - PrgEnv-gnu/8.6.0
+      - craype/2.7.33
+      - cray-dsmml/0.3.0
+      - cray-mpich/8.1.31
+      - cray-libsci/24.11.0
+      - gcc-native/13.2
+      - craype-x86-trento
+      - cray-pmi/6.1.15
+      - libfabric
+      - xpmem
+"""
+
+    def rocm_config(self):
+        return f"""\
+packages:
+  comgr:
+    buildable: false
+    externals:
+    - prefix: /opt/rocm-{self.rocm_version}
+      spec: comgr@{self.rocm_version}
+  hip:
+    buildable: false
+    externals:
+    - prefix: /opt/rocm-{self.rocm_version}
+      spec: hip@{self.rocm_version}
+    extra_attributes:
+        compilers:
+          hip: /opt/rocm-{self.rocm_version}/bin/hipcc --verbose
+        modules:
+        - rocm/{self.rocm_version}
+  hip-rocclr:
+    buildable: false
+    externals:
+    - prefix: /opt/rocm-{self.rocm_version}
+      spec: hip-rocclr@{self.rocm_version}
+  hipblas:
+    buildable: false
+    externals:
+    - prefix: /opt/rocm-{self.rocm_version}
+      spec: hipblas@{self.rocm_version}
+  hipcub:
+    buildable: false
+    externals:
+    - prefix: /opt/rocm-{self.rocm_version}
+      spec: hipcub@{self.rocm_version}
+  hipfft:
+    buildable: false
+    externals:
+    - prefix: /opt/rocm-{self.rocm_version}
+      spec: hipfft@{self.rocm_version}
+  hipify-clang:
+    buildable: false
+    externals:
+    - prefix: /opt/rocm-{self.rocm_version}
+      spec: hipify-clang@{self.rocm_version}
+  hipsparse:
+    buildable: false
+    externals:
+    - prefix: /opt/rocm-{self.rocm_version}
+      spec: hipsparse@{self.rocm_version}
+  hsa-rocr-dev:
+    buildable: false
+    externals:
+    - prefix: /opt/rocm-{self.rocm_version}
+      spec: hsa-rocr-dev@{self.rocm_version}
+  hsakmt-roct:
+    buildable: false
+    externals:
+    - prefix: /opt/rocm-{self.rocm_version}
+      spec: hsakmt-roct@{self.rocm_version}
+  llvm-amdgpu:
+    buildable: false
+    externals:
+    - prefix: /opt/rocm-{self.rocm_version}/lib/llvm
+      spec: llvm-amdgpu@{self.rocm_version}
+  rccl:
+    buildable: false
+    externals:
+    - prefix: /opt/rocm-{self.rocm_version}
+      spec: rccl@{self.rocm_version}
+  rocalution:
+    buildable: false
+    externals:
+    - prefix: /opt/rocm-{self.rocm_version}
+      spec: rocalution@{self.rocm_version}
+  rocblas:
+    buildable: false
+    externals:
+    - prefix: /opt/rocm-{self.rocm_version}
+      spec: rocblas@{self.rocm_version}
+  rocfft:
+    buildable: false
+    externals:
+    - prefix: /opt/rocm-{self.rocm_version}
+      spec: rocfft@{self.rocm_version}
+    variants: amdgpu_target=gfx90a amdgpu_target_sram_ecc=gfx90a
+  rocm-clang-ocl:
+    buildable: false
+    externals:
+    - prefix: /appl/lumi/SW/CrayEnv/EB/rocm/{self.rocm_version}
```

Comment:
@hagertnl please check this path, it has "lumi" in it.

Reply:
Fixed.